### PR TITLE
reorganize cartography section

### PR DESCRIPTION
**Pull request by ramiroaznar**
cc @jsanz

### CODE REVIEW

File: 03-cartography/README.md [hunk 1]
```diff
@@ -1,43 +1,78 @@
...
 
-## Agenda
+1. [Cartographic Design](#design)
+   - [Talks](#talks)
+   - [CartoCSS](cartocss)
+   - [TurboCARTO](#turbo)
+2. [Exercises](#design)
+3. [Resources](#design)
+   - [Colors](#design)
+   - [Other resources](#resources)
```
Comment (jsanz) [inline 03-cartography/README.md]: I'd remove an index that points to just a list of links.

File: 03-cartography/exercises/turbo.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+  * [Building](#buildings)
+  * [Opacity](#opacity)
+  * [Lines](#lines)
+  * [Cartocolors](#cartocolors)
+* [RESOURCES](#resources)
+
+<hr>
+
+<a name="turbo"></a>
+### CARTOCSS vs. TURBO CARTO
```
Comment (jsanz) [inline 03-cartography/exercises/turbo.md]: This needs a bit of some introductory wording, what is the objective of this content? anything you should know beforehand? what will I learn? why?

**Review by jsanz — CHANGES_REQUESTED** [review]
It's looking OK in general, just a couple things I think you can improve.

File: 03-cartography/README.md (hunk 1 above)
Comment (ramiroaznar) [reply 03-cartography/README.md]: Removed.

File: 03-cartography/exercises/turbo.md (hunk 2 above)
Comment (ramiroaznar) [reply 03-cartography/exercises/turbo.md]: I added some wording explaining the purpose of this workshop.

**Pull request merged**